zones/miraheze.wiki

```diff
@@ -27,9 +27,8 @@ $ORIGIN miraheze.wiki.
 ; Servers
 
 ; Services
-status		               CNAME	stats.uptimerobot.com.
 phab		               CNAME	phabricator.miraheze.org.
-status3.miraheze.wiki.         CNAME   miraheze.hund.io.
+status                         CNAME    miraheze.hund.io.
```

Comment:
keep it as status.miraheze.wiki i think that fixed the cert error anyways.

Reply:
really? something that has zero impact fixed it?